[backend] misc changes

fenrir-z opened the pull request:
* cmd filter: remove -p and -P (now uses --cis and --ex-cis), also change invoker task / cmd filter
* cmd mining: --add-annotations change to --add-prediction, also change invoker task mining
* viz get_datastet_info api: {negative_info: {negative_images_cnt: 5, project_negative_imgs_cnt: 5}} change to {negative_images_cnt: 5}
* update associated test cases
* viz: remove class_names_count and new_types
* viz: negative_images_count -> negative_assets_count
* viz: remove positive_images_count
* viz: total_images_count -> total_assets_count
* viz: cnt -> count, images -> assets

Commit:
cmd filter: remove -p and -P;
cmd mining: --add-annotations -> --add-prediction;
viz get_dataset_info api: modify negative_images_cnt structure

Commit:
viz: update test case

phoenix-xhuang committed:
Merge branch 'dev' into zzw/fix-cmd-args

phoenix-xhuang committed:
Merge branch 'dev' into zzw/fix-cmd-args

IJtLJZ8Rm4Yr commented on ymir/backend/src/ymir_viz/src/controllers/dataset_controller.py:
```diff
@@ -28,10 +28,7 @@ def get_dataset_info(user_id: str, repo_id: str, branch_id: str) -> DatasetResul
-            "negative_images_cnt": 0,
-            "project_negative_images_cnt": 0,
-        },
+        "negative_images_cnt": 0,
```
要不要统一用 negative_images_count，我看到下面又有 `negative_images_cnt` 又有 `negative_images_count`，注释里面也要更新一下

fenrir-z replied:
updated

phoenix-xhuang committed:
Merge branch 'dev' into zzw/fix-cmd-args

Commit:
in mir_storage_ops and ymir viz apis:
remove class_names_count and new_types;
negative_images_count -> negative_assets_count;
remove positive_images_count;
total_images_count -> total_assets_count;
cnt -> count
